packages/php/blueprint/src/ResourceStorages/LocalPluginResourceStorage.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Automattic\WooCommerce\Blueprint\ResourceStorages;
+
+class LocalPluginResourceStorage implements ResourceStorage {
+	protected array $paths   = array();
+	protected string $suffix = 'plugins';
+	public function __construct( $path ) {
+		$this->paths[] = $path;
+	}
+
+	/**
+	 * Local plugins are already included (downloaded) in the zip file.
+	 * Return the full path.
+	 *
+	 * @param $slug
+	 *
+	 * @return false|string
+	 */
+	public function download( $slug ): string {
```

Comment:
```suggestion
	public function download( $slug ): false|string {
```

Reply:
@chihsuan Unfortunately, multiple return type is only available in PHP 8+. Since WC still support 7.4, we can't use it 😞

Reply:
Update: Looked into it a bit more and confirmed that we can use `?string` instead.  

Updated in [Correct function return types](https://github.com/woocommerce/woocommerce/pull/49763/commits/69ccfedb294798844546ff9e0910042adb706d87)